<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphs</class>
 <widget class="QMainWindow" name="Graphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphs_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="add_vert_btn">
       <property name="text">
        <string>Добавить вершину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_vert_btn">
       <property name="text">
        <string>Удалить вершину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_rib_btn">
       <property name="text">
        <string>Создать ребро</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_rib_btn">
       <property name="text">
        <string>Удалить ребро</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bfs_btn">
       <property name="text">
        <string>Обход  в ширину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dfs_btn">
       <property name="text">
        <string>Обход  в глубину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="travel_btn">
       <property name="text">
        <string>Метод ветвей и границ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="question_btn">
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="info_line">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>470</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>440</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поле для ввода дополнительной информации:</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
